README.md

```diff
@@ -9,10 +9,12 @@ KEA will allow functionality to specify which of the columns in the file should
 
 Python 3.10 or above.
 Packages in requirements.txt
-
+git
```

Comment:
Very nitpicky request but the requirements aren't showing on newlines so it looks like we're saying requirements.txt is part of git, or looks off at the least. Adding newlines would help.